src/fastoad/models/performances/mission/routes.py

```diff
@@ -74,7 +75,7 @@ def cruise_distance(self):
 
     @cruise_distance.setter
     def cruise_distance(self, cruise_distance):
-        self.cruise_segment.target.ground_distance = float(cruise_distance)
+        self.cruise_segment.target.ground_distance = scalarize(cruise_distance)
```

Comment:
Is it correct to proceed this way ? Because if for some reasons cruse_distance is an array of more than one element, you'll get an array in return while it looks like you need a float.

Reply:
Well, if `cruise_distance` is an array, I want the program to crash because there is no reason for that.
Actually, the best option here would be to have `scalarize` raise an error for a multiple-element array. But I want `scalarize` to be fast, so I'm not fond of adding a new `if` in it...

Reply:
Can't the try ... except ... be used for that ?

Reply:
Actually, it's simpler than that 😅. I just have to do: 
```suggestion
        self.cruise_segment.target.ground_distance = np.asarray(cruise_distance).item()
```
 `scalarize()` is here to process the generic case where the input is "authorized" to be an n-elements array. It simply does not apply here.